rfcs/0057-named-ellipses.md

```diff
@@ -0,0 +1,36 @@
+---
+feature: Named Ellipses OR function argument consistency
+start-date: 2019-11-14
+author: deliciouslytyped
+co-authors: none
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues:
+  Named Ellipses - https://github.com/NixOS/nix/issues/2998
+---
+
+# Summary
+[summary]: #summary
+
+It should be possible to bind a name to ellipses in a function definition like `{ a, ...@extra }: null`, and `{ a, extra@... }: null`. This makes intuitive sense, and could remove the need for a lot of uses of `removeAttrs` that really just want to refer to the contents of ellipses.
+
+TODO: The latter could be substantiated.
+
+# Detailed design
+[design]: #detailed-design
```

Comment:
It would be useful to have an example of how people are currently using `removeAttrs` somehwere in the RFC, for comparison.

Reply:
Yeah that's what I meant. I'll add something when I have some spare brain for it, unless someone else wants to give it a shot.